internal/testhelper/testhelper.go

```diff
@@ -18,14 +21,19 @@ func FindTestData(t *testing.T, dataType string, dataPath string) map[string]str
 	t.Helper()
 	root := os.Getenv("GENERATE_DATA_SOURCE")
 	if root == "" {
-		t.Skip("test data: GENERATE_DATA_SOURCE environment variable not set")
+		t.Fatal("test data: GENERATE_DATA_SOURCE environment variable not set")
 	}
 
-	testData, err := AbsDataPath(os.Getenv("GENERATE_DATA_SOURCE"), dataPath)
+	_, err := os.Stat(root)
 	if err != nil {
-		t.Fatal(err)
+		t.Fatal("test data: GENERATE_DATA_SOURCE environment variable not set to an existing base path.\nSet this variable to the local path of the google-cloudevents repository.")
```

Comment:
This variable is used for multiple things. It's only the validation testing that needs the path to be absolute, so I thought it would be helpful to folks finding success running the generator with a relative path without difficulty to understand there's something else to do.

Reply:
Follow-up discussion with @muncus , I answered the wrong question.

- `Stat` ensure the configuration of the tests is correct. If it fails we fail out of the tests.
- `ReadDir` loads testdata if there are any. If it fails we skip running tests and continue.